alchemy_cms.gemspec

```diff
@@ -39,7 +39,6 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'request_store',                    ['~> 1.2']
   gem.add_runtime_dependency 'responders',                       ['~> 2.0']
   gem.add_runtime_dependency 'sass-rails',                       ['~> 5.0']
```

Comment:
We also should remove the `sass-rails` dependency, so one can more easily replace it with `sassc-rails` in their projects.